src/lib/logger.js

```diff
@@ -1,4 +1,9 @@
+import set from 'lodash.set'
+
 import minilog from '@cozy/minilog'
+import flag from 'cozy-flags'
+
+set(window, `cozy.debug.flagship`, () => flag('flagship.debug', true))
```

Comment:
did you test what happen when cozy is empty? When cozy.debug is empty? when flagship is defined to a wrong value? 

could we document these behaviours with unit tests 🙏 ?

Reply:
well lodash will handle that, that's in part why I don't think tests are needed here (lodash is already tested)